quesma/quesma/router.go

```diff
@@ -58,6 +59,41 @@ func ConfigureRouter(cfg *config.QuesmaConfiguration, sr schema.Registry, lm *cl
 	// So, if you add multiple handlers with the same path, the first one will be used, the rest will be redirected to the elastic cluster.
 	// This is current limitation of the router.
 
+	router.Register(routes.ExecutePainlessScriptPath, and(method("POST"), matchAgainstIndexNameInScriptRequestBody(tableResolver)), func(ctx context.Context, req *mux.Request) (*mux.Result, error) {
```

Comment:
This needs to be added to `routerV2.go` as well

Reply:
Done.